### PR TITLE
fix(vertical-nav): vertical nav accesibility

### LINKED ISSUE #3575: Vertical nav doesn't convey expanded state
## Describe the bug

The nav group trigger buttons do not convey their expanded/collapsed states. 

## How to reproduce

https://v1.clarity.design/vertical-nav/icon-links/normal

- Using a screen reader go to https://v1.clarity.design/vertical-nav/icon-links/normal
- Read any of the nav group trigger buttons and observe that the screen reader does not announce the expanded or collapsed state of these controls.

## Expected behavior

Use aria-expanded="true" and aria-expanded="false" to indicate the expanded and collapsed state of these controls

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

_Add any other notes about the problem here._

### PR DESCRIPTION
**Pull request by andreiconstantinbytex**
Closes #3575, #3574

Signed-off-by: Andrei Constantin <andrei.constantin@bytex.ro>

## PR Checklist

Please check if your PR fulfills the following requirements:

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

* [x] Bugfix
* [ ] Feature
* [ ] Code style update (formatting, local variables)
* [ ] Refactoring (no functional changes, no api changes)
* [ ] Build related changes
* [ ] CI related changes
* [ ] Documentation content changes
* [ ] clarity.design website / infrastructure changes
* [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #3575 #3574 

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

### CODE REVIEW

**Comment by bdryanovski** [comment]
@andreiconstantinbytex thanks for the PR, could you provide and a unit test with the change.
Also, could you add here as well https://v2.clarity.design/internationalization

Thanks :)

File: src/clr-angular/utils/i18n/common-strings.default.ts [hunk 1]
```diff
@@ -62,4 +62,6 @@ export const commonStringsDefault: ClrCommonStrings = {
   daypickerSRCurrentDecadePhrase: 'The current decade is {DECADE_RANGE}',
   // Stack View
   stackViewChanged: 'Value changed.',
+  // Vertical Nav
+  verticalNavToggle: 'Toggle Vertical Nav',
```
Comment (gnomeontherun) [inline src/clr-angular/utils/i18n/common-strings.default.ts]: We don't want to say this, it wouldn't necessarily make sense. It should probably just be `Toggle navigation group`.

Comment (andreiconstantinbytex) [reply src/clr-angular/utils/i18n/common-strings.default.ts]: I'm currently experiencing some issues with the environment, again. I'll update this PR as soon as I solve these issues.

**Comment by andreiconstantinbytex** [comment]
This PR is obsolete because we changed the forked repo from a individual one to company one, to better keep track of changes across devs. The new PR can be found here: https://github.com/vmware/clarity/pull/3860

**Comment by Jinnie** [comment]
Closing in favor of the linked PR.
Though I'd prefer when we have reviewed PRs we would finish them as they are and then move to a new fork, etc. But I see you had some issues with this one.

**Pull request closed without merging**